src/utils/streams/index.js

```diff
@@ -0,0 +1,54 @@
+const parser = require('./parser').createParser();
+const args = parser.parseArgs();
+
+const errorMsgs = require('./errorMsgs');
+const actions = require('./actions');
+
+analyzeArgs(args);
+
+function analyzeArgs (args) {
+    if (process.argv[2] === '--help' || process.argv[2] === '-h' || process.argv.length == 2) {
+        parser.printHelp();
+    } else if (args.action && (process.argv[2].indexOf('--action=') == 0 || process.argv[2].indexOf('-a=') == 0)) {
+        checkAction(args);
```

Comment:
pass only action name, not the whole `args`

Reply:
Would be better to use global args inside the checkAction?